packages/coverage-istanbul/src/provider.ts

```diff
@@ -13,7 +13,8 @@ import libCoverage from 'istanbul-lib-coverage'
 import libSourceMaps from 'istanbul-lib-source-maps'
 import { type Instrumenter, createInstrumenter } from 'istanbul-lib-instrument'
 
-// @ts-expect-error missing types
+// eslint-disable-next-line ts/prefer-ts-expect-error
+// @ts-ignore missing types
```

Comment:
These ignore comments are no longer needed as types are present. Maybe rebase from `main` helps.

https://github.com/vitest-dev/vitest/blob/e2c29eaf0158aa7c9b6d6e8e750fd24f70e65420/packages/coverage-istanbul/src/provider.ts#L10-L12

Reply:
This is v1, we didn't have types back then 🤔

Reply:
Ahh okey I see. Feel free to change the lines as did 😄